contrib/sacrebleu/setup.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License
+# is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+A setuptools based setup module.
+
+See:
+- https://packaging.python.org/en/latest/distributing.html
+- https://github.com/pypa/sampleproject
+"""
+
+# Always prefer setuptools over distutils
+from setuptools import setup
+import sacrebleu
```

Comment:
you should not include scarebleu here. This is actually why Travis is failing. In python 3.4 `typing` is a library that needs to be installed whereas in python 3.5+ it comes shipped with python. Now importing scarebleu in setup.py will try to import the `typing` module, which will only be installed once setup.py from sacrebleu has finished running. Hope that makes sense.

You will probably need to do a similar version regex dance as we do in the Sockeye setup.py:
https://github.com/awslabs/sockeye/blob/master/setup.py#L26

Reply:
Thank you!